PowerEditor/src/WinControls/ToolBar/ToolBar.cpp

```diff
@@ -26,6 +26,8 @@ using namespace std;
 
 constexpr DWORD WS_TOOLBARSTYLE = WS_CHILD | WS_VISIBLE | WS_CLIPCHILDREN | WS_CLIPSIBLINGS | TBSTYLE_TOOLTIPS | TBSTYLE_FLAT | CCS_TOP | CCS_NOPARENTALIGN | CCS_NORESIZE | CCS_NODIVIDER;
 
+const std::wstring TOOLBAR_CONFIG_EXAMPLE_FILE = L"toolbarButtonVisibility_example.xml";

```

Comment:
By asking to have "toolbarButtonVisibility_example.xml", I mean that this file should be provide as (in the same directory):
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/tabContextMenu_example.xml

and it will be installed by Notepad++ installer, so if user want to hide any icon, they can modify the content of "toolbarButtonVisibility_example.xml", and rename it to "toolbarButtonVisibility.xml", then restart Notepad++.

In this PR, only the name "toolbarButtonVisibility.xml" should be used - so if it exists, it will be loaded. If the load of "toolbarButtonVisibility.xml" is successful, it'll be processed, otherwise, it's ignored completely.